mw-webapp/src/dataAccessLogic/getUsersPreview.ts

```diff
@@ -0,0 +1,28 @@
+import {UserDTOToUserPreviewConverter} from "src/dataAccessLogic/DTOToPreviewConverter/userDTOToUserPreviewConverter";
+import {getWaysPreview} from "src/dataAccessLogic/getWaysPreview";
+import {UserPreview} from "src/model/businessModelPreview/UserPreview";
+import {WayPreview} from "src/model/businessModelPreview/WayPreview";
+import {UserService} from "src/service/UserService";
+
+const FIRST_INDEX = 0;
```

Comment:
useless

Reply:
Added Todo comment

Reply:
why ? it is easier to remove it than to create a comment, please get rid of it